ci/cpu/libcuspatial/build_libcuspatial.sh

```diff
@@ -0,0 +1,5 @@
+set -e
+
+echo "Building libcuspatial"
+conda build conda/recipes/libcuspatial --python=$PYTHON
```

Comment:
Does this depend on a Python version? It shouldn't as far as I know:
```suggestion
conda build conda/recipes/libcuspatial
```

Reply:
I believe it actually does for now. libcuspatial builds libnvstrings, nvstrings, libcudf and cudf as part of its conda build.

Reply:
With #39 it won't 😄